nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -785,11 +826,48 @@ where
         }
     }
 
+    #[expect(clippy::allow_attributes_without_reason)]
+    #[expect(clippy::type_complexity)]
+    async fn validate_received_block(
+        block: &Block<ClPool::Item, DaPool::Item>,
+        relays: &CryptarchiaConsensusRelays<
+            BlendAdapter,
+            BS,
+            ClPool,
+            ClPoolAdapter,
+            DaPool,
+            DaPoolAdapter,
+            NetAdapter,
+            SamplingBackend,
+            SamplingRng,
+            Storage,
+            TxS,
+            DaVerifierBackend,
+            RuntimeServiceId,
+        >,
```

Comment:
Can we extract this type into an alias?

Reply:
I don't think associated type alias are supported yet in stable.
> Feature `inherent associated types` is not available in the current Rust toolchain.
If there's an alternative I'm all ears :)